### ISSUE TITLE
Bug: Parameter renaming given a try/catch block with the same parameter name

### ISSUE THREAD

**Issue by notlee** [swc-project/swc#788]
**Describe the bug**

Given:
- an event listener with a function that has a parameter `e`
- a try/catch block which wraps the body of the function
- where the catch block also has a parameter `e`
- `swc` renames the function parameter `e1`

**Input code**
```
window.addEventListener('message', (e) => {
    try {
        console.log(e.data);
    } catch(e) {
        console.log(e);
    }
});
```

**Output code**
```
window.addEventListener('message', function(e1) {
    try {
        console.log(e.data);
    } catch (e) {
        console.log(e);
    }
});
```

**Config**
None. Used `--no-swcrc`.

**Expected behavior**
`swc` does not rename the event listener function parameter `e` to `e1`.

**Version**
```
@swc/cli: 0.1.24
@swc/core: 1.1.45
```

**Additional context**
I spotted these issues related to variable renaming. I'm unsure if they are related:
- https://github.com/swc-project/swc/issues/287
- https://github.com/swc-project/swc/issues/405

**Comment by notlee** [comment]
Great work @kdy1! Thanks for closing this so quickly

**Comment by kdy1** [comment]
@notlee Thanks! I'm currently deploying v1.1.46. Note that it would take some time as swc provides prebuilt binaries for 5 x 3 use cases.

**Comment by kdy1** [comment]
Published v1.1.46
